hack/update-manifests.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
I also ran this through the [shellcheck](shellcheck.net) and here is the list of suggestions:
```
$ shellcheck myscript
 
Line 22:
cd hack/manifest-gen/
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: (apply this, apply all SC2164)
cd hack/manifest-gen/ || exit
 
Line 28:
cd -
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
^-- SC2103: Use a ( subshell ) to avoid having to cd back.

Did you mean: (apply this, apply all SC2164)
cd - || exit
 
Line 33:
    KIND=`yq r $f kind`
         ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
               ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2006, apply all SC2086)
    KIND=$(yq r "$f" kind)
 
Line 37:
        NAME=`yq r $f spec.names.kind`
             ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
                   ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2006, apply all SC2086)
        NAME=$(yq r "$f" spec.names.kind)
 
Line 58:
NAME=`yq r $f metadata.name`
     ^-- SC2006: Use $(...) notation instead of legacy backticked `...`.
           ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2006, apply all SC2086)
NAME=$(yq r "$f" metadata.name)
 
Line 60:
if [ ! -z "$KIND" ]
     ^-- SC2236: Use -n instead of ! -z.
 
Line 68:
cp $f config/manifests/$NAME
   ^-- SC2086: Double quote to prevent globbing and word splitting.
                       ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
cp "$f" config/manifests/"$NAME"

$
```